contrib/python-dbus-next/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "python-dbus-next"
+pkgver = "0.2.3"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+depends = ["python"]
+checkdepends = ["python-pytest", "python-pytest-asyncio"]
+pkgdesc = "DBus library for python"
+maintainer = "Gnarwhal <git.aspect893@passmail.net>"
+license = "MIT"
+url = "https://pypi.org/project/dbus-next"
+source = f"https://github.com/altdesktop/python-dbus-next/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "db19689b0de50edd8587e8b55fcc6c30fe5155d813b9972e152ee05790beef59"
+# no dbus session bus in chroot
```

Comment:
maybe we can add `make_check_wrapper` support to `python_pep517` so we can just do `make_check_wrapper = ["dbus-run-session"]` + `dbus` in `checkdepends` for this

Reply:
err, it's already supported, wtf was i thinking